velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -68,7 +68,7 @@ class ParquetTpchTest : public testing::Test {
     connector::registerConnector(tpchConnector);
 
     saveTpchTablesAsParquet();
-    tpchBuilder_->initialize(tempDirectory_->getPath());
+    tpchBuilder_->initialize(tempDirectory_->getPath(), 1);
```

Comment:
Can we use a default value of 1 for `parquetPrefetch` so this change can be reverted?

Reply:
Ok sounds good!